### ISSUE TITLE
Bind a Keybinding to another Keybinding

### ISSUE THREAD

**Issue by Jaimeloeuf** [microsoft/vscode#98578]
*I tried to google this but nothing came out of it and looking at the extension's API this does not seem possible yet.

tl;dr Something like xmodmap or similiar to map a key to another key BUT only within vs code.  

Feature description in point form
- Is it possible to have an API or shortcut to allow us to bind a keybinding to another keybinding?
- Right now, when you make a key binding, either through settings directly or using an extension, you always bind a key to a command. Where a command is a executable vs code action. Would I be able to bind to another keybinding instead?
- If I wanted to change it I have to change that single keybinding, and if the key I am trying to replace is used in many places (e.g. Escape), it is extremely tedious to change them all.
- So for example, I want to create a keybinding for \<Alt + e\> to \<Escape\>
    - This way, I can prevent modifying all the keybindings that already use Escape with 1 additional keybinding / mapping

Purpose of this and who it serves:
- Ease of use / remapping for users
- For when a key is broken on some people's keyboard
- Missing key due to non standard / non english keyboards

An example usage would be like [this extension](https://github.com/Enkel-Digital/rsi-free) where instead of being able to map \<Alt + e\> to \<Escape\> directly, this extension manually adds the new key binding for every single command where \<Escape\> triggers.

**Comment by Jaimeloeuf** [comment]
In fact this can be extended to allow for combo key bindings, e.g. where 1 key binding can execute commands of 2 or more keybindings. Allowing far more complex use shortcut cases
